### ISSUE TITLE
UX: Pause units and structures post game

### ISSUE THREAD

**Issue by Microhive** [dtchitt/wc3-risk-system#8]
# Issue/Observation
I have observed that unit production, movement, and engagements continue in the post game phase where a player has won. It would be a better casting and watching experience if all the noise would stop. 

Having said that, this is not necessarily something that needs to be done. For now, I am just documenting potential small UX improvements.

# Proposal
My proposal is to pause all units and structures once a player has won. 

# Notes

Pause documentation here: https://cipherxof.github.io/w3ts/docs/api/classes/unit#paused

**Comment by Microhive** [comment]
@dtchitt merge the solution.

**Comment by Microhive** [comment]
Ops, closed wrong solution.

**Comment by dtchitt** [comment]
No worries, I noticed you linked the w3ts docs for pause infomation, I do try to avoid the w3ts classes. I used them in a prior iteration of the project and noticed a decrease in fps when relying on them on. Here is information for native warcraft 3 types and definitions  [common.j](https://github.com/cipherxof/war3-types/blob/master/scripts/common.j) [blizzard.j](https://github.com/cipherxof/war3-types/blob/master/scripts/blizzard.j)
For the most part, we will use stuff from common.j. blizzard.j is blizzard functions that are built around common.j, they are known to be not that great and can cause issues in some cases
